packages/types/src/company.ts

```diff
@@ -4,10 +4,10 @@ export const CompanySchema = z.object({
   id: z.string().uuid(),
   createdAt: z.date(),
   name: z.string().max(50),
-  description: z.string(),
+  description: z.string().nullable(),
```

Comment:
This should probably also be required because it will be shown on the company pages

Reply:
agreed